### ISSUE TITLE
add missing disciplines

### ISSUE THREAD

**Issue by mbjones** [NCEAS/adc-disciplines#1]
When using this taxonomy for our advisory board recruitment, it became clear that some key disciplines are missing, including:

- Cyberinfrastructure
- Data science

Let's re-evaluate this list and then put it into action in our dataset classifications. cc @jeanetteclark

**Comment by mbjones** [comment]
Added some key disciplines, and rearranged the hierarchy. After review, this is probably a good start.

**Comment by mbjones** [comment]
Added `Evolution`.

**Comment by jeanetteclark** [comment]
after evaluating, we seem short on a couple of fields...

`glaciology` and `climatology` seem like they can fit in relatively easily under `geosciences`.

There are a handful of other types of datasets that I'm not sure how we would categorize. We have lots of snow cover/snow depth datasets. Where would those fall do you think? Related would be ice cover and albedo. Maybe these fall under physical geography?

**Comment by mbjones** [comment]
Thanks @jeanetteclark . I did another pass, currently tagged as v0.4.0 where I added disciplines, simplified naming, added missing disciplines, and consolidated. I also restructured the tree to follow the class hierarchy used in wikipedia, with Natural Science as a superclass of Physical Science and Life Science. I also added wikipedia's Formal Science tree as a reasonable place for math, stats, etc.

Check out the comparison and notes in InVision: https://projects.invisionapp.com/freehand/document/3RzzF046p
